sql.go

```diff
@@ -88,7 +88,13 @@ AND   job_id    = $7::bigint
 INSERT INTO que_jobs
 (queue, priority, run_at, job_class, args, shard_id)
 VALUES
-(coalesce($1::text, ''::text), coalesce($2::smallint, 100::smallint), coalesce($3::timestamptz, now()::timestamptz), $4::text, coalesce($5::json, '[]'::json), $6::uuid)
+(coalesce($1::text, ''::text), coalesce($2::smallint, 100::smallint), coalesce($3::timestamptz, now()::timestamptz), $4::text, coalesce($5::jsonb, '[]'::jsonb), $6::uuid)
+`
+
+	sqlCondInsertJob = sqlInsertJob + `
+ON CONFLICT 
+($7)
```

Comment:
This must be a unique constraint, but you will note that there are none in the table schema. This is for the same reasons I mentioned in my other comment - keeping this library pure of the `analytics_etl` stuff, and letting it pile on the extra indexes and constraints while providing some way to support it down here.

Again, don't love it, but it feels like the best separation of concerns.